source/database-connections/mongodb.txt

```diff
@@ -0,0 +1,205 @@
+.. _rm-connect-to-mongo:
+
+=====================
+Connect to MongoDB
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page shows how to save a new MongoDB connection from the :guilabel:`Connections` 
+tab. You can use a saved connection for any task that requires connecting 
+to a database.
+
+.. note::
+
+   Relational Migrator does not support MongoDB to MongoDB migrations. For information 
+   on supported databases, see :ref:`<supported-databases>`.
+
+About this Task
+---------------
+
+To save a MongoDB connection in Relational Migrator, you must:
+
+- Identify your database connection string or database information.
+
+- Connect to MongoDB.
+
+.. important::
+
+   To plan your migration and map your relational schema to a 
+   MongoDB schema, you must connect using an account with the relevant 
+   ``read`` permissions.
+   
+   To migrate data, you must connect to MongoDB using a service account with 
+   the ``readWrite`` role. For information about creating a MongoDB user with 
+   the required permisions, see :ref:`<rm-mongodb-service-user>`.
+
+Before you Begin
+----------------
+
+To save a connection to a MongoDB Atlas cluster, you must be 
+:ref:`logged in <rm-atlas-log-in>` with MongoDB Atlas.
+
+Steps
+-----
+
+.. _rm-mongo-connection-string:
+.. _rm-mongodb-database-connection-strings:
+
+Database Connection String
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Relational Migrator supports all :manual:`MongoDB connection string options
+</reference/connection-string-options/>` except :urioption:`appName`. It
+overrides ``appName`` when connecting to your MongoDB deployment.
+
+Atlas Connection URI
+''''''''''''''''''''
+
+.. code-block::
+
+   mongodb+srv://username:password@clusterurl.mongodb.net/database
+
+For example, to use an account named ``migrator-service`` with password
+``hunter2`` to connect to the ``MongoEnterprises`` database:
+
+.. code-block::
+
+   mongodb+srv://migrator-service:hunter2@cluster1.abc123.mongodb.net/MongoEnterprises
+
+On-Premises Connection URI
+''''''''''''''''''''''''''
+
+.. code-block::
+
+   mongodb://username:password@host:port/database
+
+For example, to use an account named ``migrator-service`` with password
+``hunter2`` to connect to the ``MongoEnterprises`` database:
+
+.. code-block::
+
+   mongodb://migrator-service:hunter2@localhost:27017/MongoEnterprises
+
+.. _rm-x509-auth:
+
+Using X.509 Authentication
+''''''''''''''''''''''''''
+
+To connect to MongoDB using X.509 authentication, omit a username and password.
+Instead, specify the on-premises host and port or the Atlas cluster URL and set 
+the following options:
+
+.. code-block::
+
+   <on-premises host:port or clusterurl.mongodb.net>?authSource=$external&authMechanism=MONGODB-X509&tlsCertificateKeyFile=/path/to/cert/client.p12&tlsCertificateKeyFilePassword=certpassword
+
+
+.. list-table::
+   :header-rows: 1
+
+   * - Option
```

Comment:
[question]

Should this table have a *Required* field? This might be a helpful addition.